cypress/integration/jsPromisesPracticeDOM.spec.js

```diff
@@ -15,73 +12,59 @@ const thirdResolvedMsg = 'Third promise was resolved';
 
 describe('Promises in DOM', () => {
   beforeEach(() => {
+    cy.clock();
     cy.visit('/');
   });
 
-  it('should resolve first promise on the left click', () => {
-    cy.clickButton({ button: 0 });
-    cy.checkMessageDisplayed(firstResolvedMsg);
-    // NOTE: waiting for reject
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(3000);
-    cy.checkMessageDisplayed(firstRejectedMsg).should('not.exist');
-  });
+  describe('firstPromise', () => {
+    it('should be resolved after the left click', () => {
+      page.body().click();
 
-  it('should resolve first promise on the right click', () => {
-    cy.clickButton({ button: 2 });
-    cy.checkMessageDisplayed(firstResolvedMsg);
-  });
+      page.notification().should('include.text', firstResolvedMsg);
+    });
 
-  it('should resolve first promise on the middle button click', () => {
-    cy.clickButton({ button: 1 });
-    cy.checkMessageDisplayed(firstResolvedMsg);
-  });
+    it('should be rejected after 3 seconds of inactivity', () => {
+      cy.tick(3000);
 
-  it('should ignore middle button click for the second promise', () => {
-    cy.clickButton({ button: 1 });
-    cy.clickButton({ button: 1 });
-    cy.checkMessageDisplayed(secondResolvedMsg).should('not.exist');
-  });
+      page.notification().should('include.text', firstRejectedMsg);
+      page.notification().should('not.include.text', firstResolvedMsg);
```

Comment:
Also we should check that message will not change to success one if we click after 3000